### PR TITLE
Document `--rpc-max-logs-range`

### LINKED ISSUE #1217: Document `--rpc-max-logs-range` CLI option
### Describe the issue

Document the `--rpc-max-logs-range` CLI option. See PR https://github.com/hyperledger/besu/pull/4597.

### What page has the issue?

.

### Issue type

Missing content

### Change suggestion

_No response_

### More context

_No response_

### Besu documentation issue guidelines

- [X] I read and followed this project's documentation guidelines.

### PR DESCRIPTION
**Pull request by alexandratran**
Document the `--rpc-max-logs-range` CLI option for `eth_getLogs`. Fixes #1217.

### CODE REVIEW

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -2787,6 +2787,43 @@ A list of comma-separated TLS protocols to support. The default is `DEFAULT_TLS_
...
+
+When using [`eth_getLogs`](../api/index.md#eth_getlogs), the maximum number of blocks to get logs
+from.
+Set to 0 to specify no limit.
+The default is 1000.
+
+!!! warning
+
+    Using `eth_getLogs` to get logs from a large range of blocks, especially an entire chain from
+    its genesis block, can cause Besu to hang and never return a response.
```
Comment (mark-terry) [inline docs/public-networks/reference/cli/options.md:2824]: "...genesis block, _**may cause Besu to hang for an indeterminable amount of time while the response is generated.**_"

Does that sound more precise?

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -2787,6 +2787,43 @@ A list of comma-separated TLS protocols to support. The default is `DEFAULT_TLS_
...
+    BESU_RPC_MAX_LOGS_RANGE=500
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    rpc-max-logs-range=500
+    ```
+
+When using [`eth_getLogs`](../api/index.md#eth_getlogs), the maximum number of blocks to get logs
```
Comment (mark-terry) [inline docs/public-networks/reference/cli/options.md:2816]: ...blocks to _**retrieve**_ logs from.

File: docs/public-networks/reference/api/index.md [hunk 3]
```diff
@@ -2637,11 +2637,11 @@ Returns an array of [logs](../../concepts/events-and-logs.md) matching a specifi
 Leave the [`--auto-log-bloom-caching-enabled`](../cli/options.md#auto-log-bloom-caching-enabled)
 command line option at the default value of `true` to improve log retrieval performance.
 
-!!! attention
-
-    Using `eth_getLogs` to get the logs from a large range of blocks, especially an entire chain from
+!!! important
+    Using `eth_getLogs` to get logs from a large range of blocks, especially an entire chain from
     its genesis block, can cause Besu to hang and never return a response.
```
Comment (mark-terry) [inline docs/public-networks/reference/api/index.md:2642]: See my comment on this text in the other file.

**Review by mark-terry — COMMENTED** [review]
Looking good - I've left a few suggestions.

File: docs/public-networks/reference/api/index.md [hunk 4]
```diff
@@ -2641,7 +2641,8 @@ command line option at the default value of `true` to improve log retrieval perf
 
 !!! important
     Using `eth_getLogs` to get logs from a large range of blocks, especially an entire chain from
-    its genesis block, can cause Besu to hang and never return a response.
+    its genesis block, may cause Besu to hang for an indeterminable amount of time while generating
```
Comment (rolandtyler) [inline docs/public-networks/reference/api/index.md:2644]:
```suggestion
    its genesis block, might cause Besu to hang for an indeterminable amount of time while generating
```

File: docs/public-networks/reference/cli/options.md [hunk 5]
```diff
@@ -2806,15 +2806,16 @@ A list of comma-separated TLS protocols to support. The default is `DEFAULT_TLS_
...
+When using [`eth_getLogs`](../api/index.md#eth_getlogs), the maximum number of blocks to retrieve
+logs from.
 Set to 0 to specify no limit.
 The default is 1000.
 
 !!! warning
 
     Using `eth_getLogs` to get logs from a large range of blocks, especially an entire chain from
-    its genesis block, can cause Besu to hang and never return a response.
+    its genesis block, may cause Besu to hang for an indeterminable amount of time while generating
```
Comment (rolandtyler) [inline docs/public-networks/reference/cli/options.md:2817]:
```suggestion
    its genesis block, might cause Besu to hang for an indeterminable amount of time while generating
```

**Review by rolandtyler — APPROVED** [review]
2 Nits, otherwise LGTM

**Review by mark-terry — APPROVED** [review]
LGTM.

**Pull request merged**